src/Compilers/CSharp/CscCore/CscCore.csproj

```diff
@@ -17,7 +17,6 @@
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <TargetFramework>netcoreapp1.1</TargetFramework>
     <RuntimeIdentifiers>win7-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;osx.10.12-x64</RuntimeIdentifiers>
-    <PackageTargetFallback>portable-net452</PackageTargetFallback>
```

Comment:
I've just copied these proj files from master, so here is the diff. 
No idea where this came from in the branch.